src/madie-madie-editor.tsx

```diff
@@ -1,7 +1,8 @@
-import React from "react";
+import React, { FC } from "react";
```

Comment:
Not sure if `React.FC` is recommended in typescript world

Reply:
We can only export a component as a FunctionComponent.